magenta/music/midi_io.py

```diff
@@ -323,7 +333,14 @@ def note_sequence_to_pretty_midi(
     if instr_id > 0:
       instrument = pretty_midi.Instrument(prog_id, is_drum)
       pm.instruments.append(instrument)
+    # propagate instrument name to the midi file
     instrument.program = prog_id
+    for (program_id, name) in instrument_names.items():
+      if program_id == instr_id:
+        instrument.name = name
+        break
+      else:
+        instrument.name = ''
```

Comment:
couldn't this just be:
```
if instr_id in instrument_names:
  instrument.name = instrument_names[instr_id]
```

Reply:
Yes. You are right. I changed it to what you suggested!